File: Programming Basics/JavaScript/Conditional Statements Advanced/Labs/working-hours.js
function solve(input) {
    const hour = Number(input[0]);
    const day = input[1];

    const isWorkingDay = day === "Monday" || day === "Tuesday" || day === "Wednesday" || day === "Thursday" || day === "Friday" || day === "Saturday";
    const isWorkingHour = 10 <= hour && hour <= 18;

    if (isWorkingDay && isWorkingHour) console.log("open");
    else console.log("closed");
}

File: Programming Basics/C++/Conditional Statements Advanced/Labs/WorkingHours/WorkingHours.cpp
#include <iostream>
using namespace std;

int main()
{
    int hour;
    string day;
    cin >> hour >> day;

    bool isWorkingDay = day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday" || day == "Saturday";
    bool isWorkingHour = 10 <= hour && hour <= 18;

    if (isWorkingDay && isWorkingHour) cout << "open" << endl;
    else cout << "closed" << endl;
}
